### DISCUSSION TITLE
posts/three-day-weekend/

### DISCUSSION THREAD

**Discussion by giscus** [Shitao5/shitao-blog#39] [Comments]
# posts/three-day-weekend/

本周五（0308）公司团建，上午徒步登山，看看蛤蟆峰、宝石山和西湖边的风景，吃完午饭便自由活动。又恰逢周五是女神节1，小王下午也放假，毕业后没有在工作日回过学校，于是发扬爱攒局的传统，开始摇人。这天也是巧，GM 考研顺利在图书馆搜罗社会学史的书；YZ 毕业后在学校计财处工作；老师 PL 说他的课差不多结束，可以去他办公室坐坐。
[&hellip;] 买了春见，回到熟悉的教学楼，PL 问起我和小王的 &hellip;

https://shitao5.org/posts/three-day-weekend/

**Comment by hongtaoh** [comment]
表示羡慕不已

**Reply by Shitao5** [reply]
嘿嘿工作后发现维护亲密关系和自我学习成长一样重要，感觉自己更 E（外向）了。
